scripts/aws.sh

```diff
@@ -0,0 +1,488 @@
+#!/bin/bash
+
+set -uo pipefail
+
+export AWS_PAGER=""
+
+err_log="err.log"
```

Comment:
This is a global variable.
```suggestion
ERRLOG=$(mktemp /tmp/aws-stack-err.XXXXXXXX.log)"
```